### ISSUE TITLE
Is pwacompat dynamic splash images feature works correctly?

### ISSUE THREAD

**Issue by gnelitsa** [GoogleChromeLabs/pwacompat#53]
I wanted to use pwacompat dynamic splash images feature but stumbled on unexpected behavior on iOS 12.4 and higher - test examples  [Emojityper](https://emojityper.com/) , [demo site](https://googlechromelabs.github.io/pwacompat/demo/) [v8.dev](https://v8.dev/) have any splash screens. Can anyone help with this?

**Comment by SuperPat45** [comment]
pwacompat is broken due to line:

```
const dim = largestSize(icon);
```

PWA and icons works fine with this fix:

```
const dim = icon.largestSize;
```

**Comment by samthor** [comment]
Hi @gnelitsa can you expand on your issue?

@SuperPat45 this was fixed in 2.0.17, which is the currently latest published version on NPM: the [2.0.16](https://unpkg.com/pwacompat@2.0.16/pwacompat.min.js) version (look for `largestSize` in that source) had the problem but it was replaced quickly [here](https://unpkg.com/pwacompat).

**Comment by gnelitsa** [comment]
@samthor right now everything works fine. Thanks
